packages/block-editor/src/components/inserter/index.js

```diff
@@ -22,6 +27,15 @@ import InserterMenu from './menu';
 import QuickInserter from './quick-inserter';
 import { store as blockEditorStore } from '../../store';
 
+// Duplicated from elsewhere.
```

Comment:
I think there is a hook `useBlockDisplayTitle` or something. Or maybe that's where this is from.